plugins/outputs/whatap/whatap_test.go

```diff
@@ -0,0 +1,64 @@
+package whatap
+
+import (
+	"net"
+	"testing"
+
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func newWhatap(addr string) *Whatap {
+	servers := make([]string, 0)
+	servers = append(servers, "tcp://"+addr)
+	w := &Whatap{
+		Timeout: config.Duration(60 * time.Second),
+		Log:     testutil.Logger{},
+		Servers: servers,
+	}
```

Comment:
I think this is too complicated, use
```suggestion
	w := &Whatap{
		Timeout: config.Duration(60 * time.Second),
		Log:     testutil.Logger{},
		Servers: []string{"tcp://"+addr},
	}
```
instead. I would also suggest to put this to the respective locations instead of making it a function...